databases/backends/mysql.py

```diff
@@ -97,9 +97,12 @@ async def acquire(self) -> None:
     async def release(self) -> None:
         assert self._connection is not None, "Connection is not acquired"
         assert self._database._pool is not None, "DatabaseBackend is not running"
-        await self._database._pool.release(self._connection)
+        await self._internal_release(self._connection)
```

Comment:
Sorry I'm confused, how is this different from the original one?

Reply:
It seems, I forgot to make changes for mysql during debugging